tests/README.md

```diff
@@ -0,0 +1,4 @@
+# `tests` folder
+
+All tests for the functions defined in the `hooks`, and `{{ cookiecutter.repo_name }}/src` folders should be stored
+here.
```

Comment:
Why should the tests for the functions in `{{ cookiecutter.repo_name }}/src` be stored here?
Sorry if I'm probably missing something and it does instead makes sense. But functions in `{{ cookiecutter.repo_name }}/src` will be created by the template user for their project and the template user will not have access to this higher-level  cookiecutter project `tests/` folder.

Reply:
Good point! This is unclear - I've removed the reference to the template tests in commit [`79b15e3`](https://github.com/ukgovdatascience/govcookiecutter/pull/10/commits/79b15e386e5c4561c6d8393f52edf25568d1c366). I've also added in two blank `conftest.py` files to ensure pytest can run all the tests (even the ones in the template) correctly, and amended `pytest.ini` so that the `docs` folders are not collected.

With the latter, this prevents an error being raised due to the immediate relative paths of both `docs/conf.py` files (I think). Tested this with a simple squaring function in `src` and associated test - works fine!

```python
# {{ cookiecutter.repo_name }}/src/main.py

def squared(x):
    return x ** 2

# {{ cookiecutter.repo_name }}/tests/test_main.py

from src.main import squared
import pytest

@pytest.mark.parametrize("test_input, test_expected", [(2, 4), (3, 9)])
def test_squared(test_input, test_expected):
    assert squared(test_input) == test_expected

```

Grateful if you could verify!

Reply:
Excellent! 

Works for me. I also apologise as I realise I had added a `conftest.py` file myself in the template root directory of my local copy in order to run the tests. That was before Christmas and I forgot I had done it. I should have remembered it and flagged it 😅

Reply:
No worries - something that @chao-xian helped me figure out this morning! Citation available as the Note in Point 6 [here](https://docs.pytest.org/en/latest/writing_plugins.html#plugin-discovery-order-at-tool-startup). Marking this as **Resolved**.